src/DiIiS-NA/D3-GameServer/GSSystem/ItemsSystem/Item.cs

```diff
@@ -1088,7 +1088,7 @@ public virtual void OnRequestUse(Player player, Item target, int actionId, World
                                 break;
                         }
 
-                        it = items[RandomHelper.Next(@base, @base + 1)];
+                        it = items[RandomHelper.Next(@base, @base + 1)]; // FIXME: this random always returns @base
```

Comment:
probably `+ 2`?

Reply:
```csharp
var rand = new Random();
rand.Next(0, 1); // will always return 0, as probably the Microsoft development thought about it for .Length

// i.e.:
rand.Next(0, someArray.Length); // so it won't have issues with arguments
```

Reply:
@iamdroppy, Looks so :)
Where we can discuss such questions? Because I found bunch of such issues where some things cannot be picked by passing - 1. Like this:
`scenes[Next(0, scenes.Length - 1)]`